README.md

```diff
@@ -79,87 +79,146 @@ make helm-chart-push
+kubectl -n fybrik-system label configmap sample-policy openpolicyagent.org/policy=rego
+while [[ $(kubectl get cm sample-policy -n fybrik-system -o 'jsonpath={.metadata.annotations.openpolicyagent\.org/policy-status}') != '{"status":"ok"}' ]]; do echo "waiting for policy to be applied" && sleep 5; done
+```
+
+### Deploy Fybrik application which triggers module
+Deploy `FybrikApplication` in `default` namespace:
 ```bash
 kubectl apply -f fybrikapplication.yaml -n default
 ```
-3.  Check if `FybrikModule` successfully deployed:
+3.  Run the following command to wait until the `status` of the `FybrikApplication` is `ready`:
 ```bash
-kubectl get FybrikApplication -n default
-kubectl describe FybrikApplication hello-world-read-module-test -n default
+while [[ $(kubectl get fybrikapplication my-notebook -n default -o 'jsonpath={.status.ready}') != "true" ]]; do echo "waiting for FybrikApplication" && sleep 5; done
 ```
 
 4.  Check if module was triggered in `fybrik-blueprints`:
 ```bash
 kubectl get blueprint -n fybrik-blueprints
-kubectl describe blueprint hello-world-read-module-test-default -n fybrik-blueprints
+kubectl describe blueprint my-notebook-default -n fybrik-blueprints
+kubectl get job -n fybrik-blueprints
 kubectl get pods -n fybrik-blueprints
 ```
-If you are using the existing `hello-world-read-module.py`, you should see this in the `kubectl logs` of the `fybrik-blueprints` Pod:
+
+If you are using the `hello-world-read-module` image, you should see this in the `kubectl logs` of your completed Pod:
 ```
-$ kubectl logs <fybrik-blueprints pod> -n fybrik-blueprints
+$ kubectl logs my-notebook-default-hello-world-read-module-5f59d75c8f-v7z7p -n fybrik-blueprints
+
 INFO:root:
 Hello World Read Module!
-INFO:root:Starting httpd server on localhost:8000
-```
+INFO:root:The avialable datasets:
 
-Then, you can do port forwarding in order to use the server by the following command:
+INFO:root:dataset name: medals-winners
 
-```bash
-kubectl port-forward <fybrik-blueprints pod> -n fybrik-blueprints 8000:8000 &
-```
+INFO:root:    format: csv
+
+INFO:root:    endpoint_url: http://winterolympicsmedals.com/medals.csv
+
+INFO:root:    action: Redact
+
+INFO:root:    transferred_columns: ['age']
+
+INFO:root:dataset name: bank
+
+INFO:root:    format: csv
+
+INFO:root:    endpoint_url: https://raw.githubusercontent.com/juliencohensolal/BankMarketing/master/rawData/bank-additional-full.csv
+
+INFO:root:    action: Redact
+
+INFO:root:    transferred_columns: ['age']
+
+INFO:root:Starting httpd server on localhost:8000
 
-If you run the following request:
-```bash
-curl -X GET localhost:8000/medals-winners
 ```
-you get the first 10 rows of the medals-winners dataset.
 
 ## Clean
 
 Run the following command to delete the fybrik application:
 ```bash
-kubectl delete FybrikApplication hello-world-read-module-test -n default
+kubectl delete FybrikApplication my-notebook -n default
```

Comment:
please add in the end reference to the cleanup section from the quick start (it removes the cluster too)

Reply:
Added Thanks